### ISSUE TITLE
v2.4 To-Do List

### ISSUE THREAD

**Issue by ThePhaseCat** [ThePhaseCat/PhaseSMP-Network-Pack#62]
v2.4 won't be out for a few months, but as there's stuff already planned for this update, I've decided to make the To-Do List already. Also, each thing listed will have it's own issue for separate tracking! This update will probably be towards August or maybe even September.

- Add arrival music for the upcoming Phase-Town Airport Shuttle Line
- Add arrival music for the upcoming North Phase-Town Shuttle Line
- Change more textures for light blocks into simple textures
- Change water textures to be clearer
- Make bookshelf's 3D

**Comment by ThePhaseCat** [comment]
v2.4 won't be out for a few months, but as there's stuff already planned for this update, I've decided to make the To-Do List already. Also, each thing listed will have it's own issue for separate tracking! This update will probably be towards August or maybe even September.

- Add arrival music for the upcoming Phase-Town Airport Shuttle Line #63 
- Add arrival music for the upcoming North Phase-Town Shuttle Line #64 
- Change more textures for light blocks into simple textures #65 
- Change water textures to be clearer #66 
- Make bookshelf's 3D #67 

Simply just added the issues that relate to this update!

**Comment by ThePhaseCat** [comment]
v2.4 won't be out for a few months, but as there's stuff already planned for this update, I've decided to make the To-Do List already. Also, each thing listed will have it's own issue for separate tracking! This update will probably be towards August or maybe even September. Due to timing reasons, some things have been moved to updates past v2.4 sadly.

- Add arrival music/TTS for Line E (End Line) #68 
- Change more textures for light blocks into simple textures #65 
- Change water textures to be clearer #66 
- Make bookshelf's 3D #67 

Simply just added the issues that relate to this update!
